File: Assignment10/Program2.c
// 2. Accept number of rows and number of columns from user and display below pattern.
// Input :  iRow = 4  iCol = 3
// Output :
// 1 2 3
// 1 2 3
// 1 2 3
// 1 2 3

#include <stdio.h>
void Pattern(int iRow, int iCol)
{
    int rowCount = 0;
    int colCount = 0;

    for (rowCount = 1; rowCount <= iRow; rowCount++)
    {
        for (colCount = 1; colCount <= iCol; colCount++)
        {
            printf("%d \t",colCount);
        }
        printf("\n");
    }
}
int main()
{
    int iValue1 = 0, iValue2 = 0;

    printf("Enter number of rows and columns");
    scanf("%d %d", &iValue1, &iValue2);

    Pattern(iValue1, iValue2);
    return 0;
}

File: Assignment38/Program2.java
// 2. Write Java program which accept String from user and display below pattern. 

// Input :  Hello 
// Output : H e l l o    
// 			H e l l    
// 			H e l   
// 			H e   
// 			H


import java.util.*;

class Program2
{
	public static void main(String[] args) 
	{
		Scanner sobj = new Scanner(System.in);
		String str;

		System.out.println("Enter String \n");
		str = sobj.nextLine();

		Pattern pobj = new Pattern();
		pobj.Pattern(str);
	}
}

class Pattern 
{ 
 	public void Pattern(String str)  
 	{   
 		int i = 0,j = 0;

 		char [] Arr = str.toCharArray();

 		for(i = 0;i <= Arr.length - 1;i++)
 		{
 			for(j = 0;j < Arr.length - i;j++)
 			{
	 			System.out.print(Arr[j]);
 			}
 			System.out.println();
 		}
 	} 
}